### ISSUE TITLE
2021-09

### ISSUE THREAD

**Issue by qiaojy19** [qiaojy19/q-Daily#10]

**Comment by qiaojy19** [comment]
# Sep - 2021

## 09.27 - 10.01
| Title                                                                                                                                              |
| -------------------------------------------------------------------------------------------------------------------------------------------------- |
| [Coding for Straggler Mitigation in Federated Learning                                                          ](https://arxiv.org/abs/2109.1522) |
| [Decentralized Deep Learning for Mobile Edge Computing: A Survey on Communication Efficiency and Trustworthiness](https://arxiv.org/abs/2108.0398) |
| [DESTRESS: Computation-Optimal and Communication-Efficient Decentralized Nonconvex Finite-Sum Optimization      ](https://arxiv.org/abs/2110.0116) |
| [Distributed Computing With the Cloud                                                                           ](https://arxiv.org/abs/2109.1293) |
| [Distributed Computing With the Cloud                                                                           ](https://arxiv.org/abs/2109.1293) |
| [FairFed: Enabling Group Fairness in Federated Learning                                                         ](https://arxiv.org/abs/2110.0085) |
| [Federated Deep Learning with Bayesian Privacy                                                                  ](https://arxiv.org/abs/2109.1301) |
| [Federated Dropout -- A Simple Approach for Enabling Federated Learning on Resource Constrained Devices         ](https://arxiv.org/abs/2109.1525) |
| [Federated Learning in ASR: Not as Easy as You Think                                                            ](https://arxiv.org/abs/2109.1510) |
| [Federated Learning over Next-Generation Ethernet Passive Optical Networks                                      ](https://arxiv.org/abs/2109.1459) |
| [Federated Self-Supervised Contrastive Learning via Ensemble Similarity Distillation                            ](https://arxiv.org/abs/2109.1461) |
| [Federated Submodel Averaging                                                                                   ](https://arxiv.org/abs/2109.0770) |
| [FederatedDeepLearningwith Bayesian Privacy                                                                     ](https://arxiv.org/abs/2109.1301) |
| [FedIPR: Ownership Verification for Federated Deep Neural Network Models                                        ](https://arxiv.org/abs/2109.1323) |
| [Fed-LAMB: Layerwise and Dimensionwise Locally Adaptive Optimization Algorithm                                  ](https://arxiv.org/abs/2110.0053) |
| [FLASHE: Additively Symmetric Homomorphic Encryption for Cross-Silo Federated Learning                          ](https://arxiv.org/abs/2109.0067) |
| [LightSecAgg: Rethinking Secure Aggregation in Federated Learning                                               ](https://arxiv.org/abs/2109.1423) |
| [Lightweight Transformer in Federated Setting for Human Activity Recognition                                    ](https://arxiv.org/abs/2110.0024) |
| [Personalized Retrogress-Resilient Framework for Real-World Medical Federated Learning                          ](https://arxiv.org/abs/2110.0039) |
| [Secure Machine Learning over Relational Data                                                                   ](https://arxiv.org/abs/2109.1480) |
| [Secure Multi-Party Computation based Privacy Preserving Data Analysis in Healthcare IoT Systems                ](https://arxiv.org/abs/2109.1433) |
| [Sequential Estimation under Multiple Resources: a Bandit Point of View                                         ](https://arxiv.org/abs/2109.1470) |
| [TinyFedTL: Federated Transfer Learning on Tiny Devices                                                         ](https://arxiv.org/abs/2110.0110) |
## 09.20 - 09.24
| Title                                                                                                                                    |
| ---------------------------------------------------------------------------------------------------------------------------------------- |
| [A DeepLearningPerspective on Connected Automated Vehicle (CAV) Cybersecurity and Threat Intelligence](https://arxiv.org/abs/2109.10763) |
| [A GenerativeFederatedLearningFramework for Differential Privacy                                     ](https://arxiv.org/abs/2109.12062) |
| [Achieving Model Fairness in VerticalFederatedLearning                                               ](https://arxiv.org/abs/2109.08344) |
| [AsySQN: Faster VerticalFederatedLearningAlgorithms with Better Computation Resource Utilization     ](https://arxiv.org/abs/2109.12519) |
| [Backdoor Attacks onFederatedLearningwith Lottery Ticket Hypothesis                                  ](https://arxiv.org/abs/2109.10512) |
| [Decentralized WirelessFederatedLearningwith Differential Privacy                                    ](https://arxiv.org/abs/2109.09142) |
| [DeSMP: Differential Privacy-exploited Stealthy Model Poisoning Attacks inFederatedLearning          ](https://arxiv.org/abs/2109.09955) |
| [Enabling Large-ScaleFederatedLearningover Wireless Edge Networks                                    ](https://arxiv.org/abs/2109.10489) |
| [FederatedFeature Selection for Cyber-Physical Systems of Systems                                    ](https://arxiv.org/abs/2109.11323) |
| [FedProc: Prototypical ContrastiveFederatedLearningon Non-IID data                                   ](https://arxiv.org/abs/2109.12273) |
| [In-network Computation for Large-scaleFederatedLearningover Wireless Edge Networks                  ](https://arxiv.org/abs/2109.10903) |
| [LocalLearningat the Network Edge for Efficient & Secure Real-Time Predictive Analytics              ](https://arxiv.org/abs/2109.12375) |
| [MixNN: Protection ofFederatedLearningAgainst Inference Attacks by Mixing Neural Network Layers      ](https://arxiv.org/abs/2109.12550) |
## 09.13 - 09.17
| Title                                                                                                                                                            |
| ---------------------------------------------------------------------------------------------------------------------------------------------------------------- |
| [A ScalableFederatedMulti-agent Architecture for Networked Connected Communication Network                                   ](https://arxiv.org/abs/2108.00506) |
| [Achieving Model Fairness in VerticalFederatedLearning                                                                       ](https://arxiv.org/abs/2109.08344) |
| [Bayesian AirComp with Sign-Alignment Precoding for WirelessFederatedLearning                                                ](https://arxiv.org/abs/2109.06579) |
| [Comfetch:FederatedLearningof Large Networks on Memory-Constrained Clients via Sketching                                     ](https://arxiv.org/abs/2109.08346) |
| [Concept Drift Detection inFederatedNetworked Systems                                                                        ](https://arxiv.org/abs/2109.06088) |
| [Decentralized WirelessFederatedLearningwith Differential Privacy                                                            ](https://arxiv.org/abs/2109.09142) |
| [Efficient-FedRec: EfficientFederatedLearningFramework for Privacy-Preserving News Recommendation                            ](https://arxiv.org/abs/2109.05446) |
| [Enforcing fairness in private federated learning via the modified method of differential multipliers                        ](https://arxiv.org/abs/2109.08604) |
| [Exploiting Heterogeneity in RobustFederatedBest-Arm Identification                                                          ](https://arxiv.org/abs/2109.05700) |
| [FastFederatedEdgeLearningwith Overlapped Communication and Computation and Channel-Aware Fair Client Scheduling             ](https://arxiv.org/abs/2109.06710) |
| [FederatedContrastiveLearningfor Decentralized Unlabeled Medical Images                                                      ](https://arxiv.org/abs/2109.07504) |
| [FederatedLearningof Molecular Properties in a Heterogeneous Setting                                                         ](https://arxiv.org/abs/2109.07258) |
| [FederatedSubmodel Averaging                                                                                                 ](https://arxiv.org/abs/2109.07704) |
| [Improving Fairness for Data Valuation inFederatedLearning                                                                   ](https://arxiv.org/abs/2109.09046) |
| [OpenFed: An Open-Source Security and Privacy GuaranteedFederatedLearningFramework                                           ](https://arxiv.org/abs/2109.07852) |
| [PersonalizedFederatedLearningfor Heterogeneous Clients with Clustered Knowledge Transfer                                    ](https://arxiv.org/abs/2109.08119) |
| [SignGuard: Byzantine-robustFederatedLearningthrough Collaborative Malicious Gradient Filtering                              ](https://arxiv.org/abs/2109.05872) |
| [Splitfedlearningwithout client-side synchronization: Analyzing client-side split network portion size to overall performance](https://arxiv.org/abs/2109.09246) |
| [SubspaceLearningfor PersonalizedFederatedOptimization                                                                       ](https://arxiv.org/abs/2109.07628) |
| [Toward EfficientFederatedLearningin Multi-Channeled Mobile Edge Network with Layerd Gradient Compression                    ](https://arxiv.org/abs/2109.08819) |
## 09.06 - 09.10
| Title                                                                                                                                          |
| ---------------------------------------------------------------------------------------------------------------------------------------------- |
| [A Blockchain basedFederatedLearningfor Message Dissemination in Vehicular Networks                        ](https://arxiv.org/abs/2109.06667) |
| [A distillation-based approach integrating continual learning and federated learning for pervasive services](https://arxiv.org/abs/2109.04197) |
| [AcceleratingFederatedLearningwith a Global Biased Optimiser                                               ](https://arxiv.org/abs/2108.09134) |
| [AMI-FML: A Privacy-PreservingFederatedMachineLearningFramework for AMI                                    ](https://arxiv.org/abs/2109.05666) |
| [An Experimental Study of Class Imbalance in Federated Learning                                            ](https://arxiv.org/abs/2109.04094) |
| [Asynchronous Federated Learning on Heterogeneous Devices: A Survey                                        ](https://arxiv.org/abs/2109.04269) |
| [Byzantine-Robust Federated Learning via Credibility Assessment on Non-IID Data                            ](https://arxiv.org/abs/2109.02396) |
| [Cost-EffectiveFederatedLearningin Mobile Edge Networks                                                    ](https://arxiv.org/abs/2109.05411) |
| [CriticalLearningPeriods inFederatedLearning                                                               ](https://arxiv.org/abs/2109.05613) |
| [Dubhe: Towards Data Unbiasedness with Homomorphic Encryption in Federated Learning Client Selection       ](https://arxiv.org/abs/2109.04253) |
| [Fair Federated Learning for Heterogeneous Face Data                                                       ](https://arxiv.org/abs/2109.02351) |
| [FedCon: A Contrastive Framework forFederatedSemi-SupervisedLearning                                       ](https://arxiv.org/abs/2109.04533) |
| [Federated Learning Beyond the Star: Local D2D Model Consensus with Global Cluster Sampling                ](https://arxiv.org/abs/2109.03350) |
| [FederatedEnsemble Model-based ReinforcementLearning                                                       ](https://arxiv.org/abs/2109.05549) |
| [FedFair: Training Fair Models In Cross-SiloFederatedLearning                                              ](https://arxiv.org/abs/2109.05662) |
| [FedMatch: Federated Learning Over Heterogeneous Question Answering Data                                   ](https://arxiv.org/abs/2108.05069) |
| [FedTriNet: A Pseudo Labeling Method with Three Players forFederatedSemi-supervisedLearning                ](https://arxiv.org/abs/2109.05612) |
| [FedZKT: Zero-Shot Knowledge Transfer towards Heterogeneous On-Device Models in Federated Learning         ](https://arxiv.org/abs/2109.03775) |
| [Generation of Synthetic Electronic Health Records Using a Federated GAN                                   ](https://arxiv.org/abs/2109.02543) |
| [Iterated Vector Fields and Conservatism, with Applications to Federated Learning                          ](https://arxiv.org/abs/2109.03973) |
| [MultimodalFederatedLearning                                                                               ](https://arxiv.org/abs/2109.04833) |
| [On Second-order Optimization Methods for Federated Learning                                               ](https://arxiv.org/abs/2109.02388) |
| [On the Initial Behavior Monitoring Issues inFederatedLearning                                             ](https://arxiv.org/abs/2109.05385) |
| [Reconfigurable Intelligent Surface Empowered Over-the-Air Federated Edge Learning                         ](https://arxiv.org/abs/2109.02353) |
| [Source Inference Attacks inFederatedLearning                                                              ](https://arxiv.org/abs/2109.05659) |
| [System Optimization in Synchronous Federated Training: A Survey                                           ](https://arxiv.org/abs/2109.03999) |
| [Toward Communication Efficient Adaptive Gradient Method                                                   ](https://arxiv.org/abs/2109.05109) |
| [Utility Fairness for the Differentially PrivateFederatedLearning                                          ](https://arxiv.org/abs/2109.05267) |
## 08.30 - 09.03
| Title                                                                                                                                  |
| -------------------------------------------------------------------------------------------------------------------------------------- |
| [A Synergetic Attack against Neural Network Classifiers combining Backdoor and Adversarial Examples](https://arxiv.org/abs/2109.01275) |
| [Asynchronous Federated Learning for Sensor Data with Concept Drift                                ](https://arxiv.org/abs/2109.00151) |
| [Communication Efficient Tensor Factorization for Decentralized Healthcare Networks                ](https://arxiv.org/abs/2109.01718) |
| [Efficient Federated Meta-Learning over Multi-Access Wireless Networks                             ](https://arxiv.org/abs/2108.06453) |
| [Energy-Efficient Massive MIMO for Serving Multiple Federated Learning Groups                      ](https://arxiv.org/abs/2108.13512) |
| [Federated Learning: Issues in Medical Application                                                 ](https://arxiv.org/abs/2109.00202) |
| [Federated Reconnaissance: Efficient, Distributed, Class-Incremental Learning                      ](https://arxiv.org/abs/2109.00150) |
| [FedKD: Communication Efficient Federated Learning via Knowledge Distillation                      ](https://arxiv.org/abs/2108.13323) |
| [FLASHE: Additively Symmetric Homomorphic Encryption for Cross-Silo Federated Learning             ](https://arxiv.org/abs/2109.00675) |
| [Ground-Assisted Federated Learning in LEO Satellite Constellations                                ](https://arxiv.org/abs/2109.01348) |
| [GRP-FED: Addressing Client Imbalance in Federated Learning via Global-Regularized Personalization ](https://arxiv.org/abs/2108.13858) |
| [GTG-Shapley: Efficient and Accurate Participant Contribution Evaluation in Federated Learning     ](https://arxiv.org/abs/2109.02053) |
| [Statistical Estimation and Inference via Local SGD in Federated Learning                          ](https://arxiv.org/abs/2109.01326) |
| [Unit-Modulus Wireless Federated Learning Via Penalty Alternating Minimization                     ](https://arxiv.org/abs/2108.13669) |
